director-configure-blobstore.html.md.erb

```diff
@@ -72,3 +72,49 @@ The Director and the Agents can use an S3 compatible blobstore. Here is how to c
         bucket_name: test-bosh-bucket
         host: objects.dreamhost.com
     ```
+
+---
+## <a id="gcs"></a> Google Cloud Storage (GCS)
+
+The Director and the Agents can use GCS blobstore. Here is how to configure it:
+
+1. Create a GCS bucket.
```

Comment:
Link to where to do this?

Also steps:
- Create a service account with [role?]
- Generate a key for the service account